services/core-api/app/api/EMLI_contacts/resources/EMLI_contact_list.py

```diff
@@ -0,0 +1,104 @@
+from flask_restplus import Resource
+from werkzeug.exceptions import BadRequest
+from app.extensions import api
+from flask import request
+from app.api.utils.custom_reqparser import CustomReqparser
+from app.api.utils.access_decorators import requires_role_edit_emli_contacts, requires_any_of, VIEW_ALL, MINESPACE_PROPONENT
+from app.api.utils.resources_mixins import UserMixin
+from app.api.EMLI_contacts.models.EMLI_contact import EMLIContact
+from app.api.EMLI_contacts.models.EMLI_contact_type import EMLIContactType
+from app.api.EMLI_contacts.response_models import EMLI_CONTACT_MODEL
+
+
+class EMLIContactListResource(Resource, UserMixin):
```

Comment:
Again same comment as on the Project Summary Resource these two classes should and can be in the same resource file.